src/components/ControlPanel.jsx

```diff
@@ -46,11 +52,21 @@ class ControlPanel extends React.Component {
   componentWillReceiveProps(nextProps) {
     this.fetchTutorial(nextProps);
 
-    if (nextProps.tutorial !== this.props.tutorial) {
+    //If there's new tutorial data, show it...
+    //...unless there's a WorkInProgress prompt in the way.
+    if (nextProps.tutorial !== this.props.tutorial && !WorkInProgress.check(this.props.user)) {
       Tutorial.checkIfCompleted(nextProps.tutorial, nextProps.user, nextProps.preferences).then((completed) => {
         if (!completed) { this.toggleTutorial(); }
       });
     }
+    
+    //Now check if the user has any work in progress.
+    console.log('+++ componentWillReceiveProps', this.props.user !== nextProps.user, WorkInProgress.check(nextProps.user));
+    if (this.props.user !== nextProps.user && WorkInProgress.check(nextProps.user)) {
+      console.log('+++ WorkInProgressPopup', nextProps.user);
```

Comment:
Were these logs simply for debugging?

Reply:
It doesn't look like the `WorkInProgressPopup` is triggered on this line in the instance that the landing page is visited first and then the user navigates to "transcribe" on the nav bar. I think that is because the user never changes (`this.props.user !== nextProps.user`) since it has already been retrieved from the initial load.

Reply:
Argh! Thanks, these `console.log()s` need to go.

And thank you for that catch - I only tested that the LoadProgress prompt works if I refresh on the Classifier page.

Possible solution: I should put a workInProgress check on _componentDidMount..._ but ONLY if user has already logged in. (Otherwise, logged-in users who refresh the page on the Classifier will accidentally trigger the anonymous user's Save Progress data.) I realise I'm kinda giving the shaft to the non-logged-in users who want to save/retrieve their progress... but then, logically, they shouldn't be able to see the Save Progress button anyway. Hmm.